libtelemetry/src/main/java/com/mapbox/services/android/telemetry/TelemetryService.java

```diff
@@ -21,19 +21,33 @@
 public class TelemetryService extends Service implements TelemetryCallback, LocationEngineListener {
   private LocationReceiver locationReceiver = null;
   private TelemetryReceiver telemetryReceiver = null;
+  private boolean isBound;
   // For testing only:
   private boolean isLocationReceiverRegistered = false;
   private boolean isTelemetryReceiverRegistered = false;
   private LocationEngine locationEngine = null;
+  private int locationPriority = LocationEnginePriority.HIGH_ACCURACY;
```

Comment:
We should use `@LocationEnginePriority.PowerMode` instead of the previous constants.